key.go

```diff
@@ -692,6 +692,7 @@ func (k *Key) PublicKey() (crypto.PublicKey, error) {
 }
 
 // PrivateKey returns a crypto.PrivateKey generated using Key's parameters.
+// Compressed point is not supported for EC2 keys.
```

Comment:
This comment seems not true.

This library does not support compressed points, but EC2 keys do.